### ISSUE TITLE
String indexing and comparison doesn't seem to work

### ISSUE THREAD

**Issue by wolfv** [mitsuhiko/minijinja#291]
## Description

On the playground, running:

```
  {{ "test" if name[0] == 'W' else "ohno" }}
  {{ "test" if nav[0].href == '/' else "ohno" }}
```

works in the second case, but not in the first case. Looks like there might be another indexing function used when in a boolean expression?

I saw this also locally with 1.0.0-alpha.

## What did you expect

I think ideally indexing into strings would also work in boolean expressions.

**Comment by mitsuhiko** [comment]
I know why this is and will fix it. It's probably that `"foo"[0]` returns a `char` and `"W"` is a `str` of one character and they are not comparing to equality.

**Comment by mitsuhiko** [comment]
Fixed in 57133043d9cb9256a4859eeb717a0b083bd50135
